SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1268,6 +1283,12 @@ void USpatialActorChannel::ServerProcessOwnershipChange()
 		bUpdatedThisActor = true;
 	}
 
+	// Owner changed, update the actor's interest over it.
+	bool bOwnerReady;
```

Comment:
Does owner ready mean - owner has an ID? If so I think it's worth being more explicit about it as I confused this with actor "Readiness" as we use it in lifecycle stuff.

Reply:
I have no opinion on that, IMO, it could be either. The way it is right now is that a PC gets an entity id before being replicated. So are we comfortable about declaring interest over an entity that's not ready in the lifecycle sense ? Maybe we are, maybe we are not.

Reply:
If we have an entity ID then yes. Interest just means you will eventually know about this thing.